cdap-unit-test/src/test/java/io/cdap/cdap/test/app/AppUsingGetServiceURL.java

```diff
@@ -107,6 +107,31 @@ public void ping(HttpServiceRequest request, HttpServiceResponder responder) thr
       }
     }
 
+    @GET
+    @Path("appWithServices")
```

Comment:
its better if this and the method name are a description of what the method does, something like 'forward'